internal/proto/query_coord.proto

```diff
@@ -199,7 +200,6 @@ message SyncNewCreatedPartitionRequest {
   common.MsgBase base = 1;
   int64 collectionID = 2;
   int64 partitionID = 3;
-  schema.CollectionSchema schema = 4;
 }
```

Comment:
Add some comments to remind others that serial number 4 has already been used.

Reply:
it's ok because schema is added into master branch and has never been released.